benchexec/tools/witnesslint.py

```diff
@@ -0,0 +1,21 @@
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
```

Comment:
This module does not override `version`, which means that users will not see in the result tables which tool version was used. This can be confusing and hinders reproducibility.

The best would be to add a parameter like `--version` to your tool and read and return the version number from the method `version`. You can call the utility method `self._version_from_tool` to do this easily.

Reply:
The linter can now provide version info via the `--version` flag -> added `version` to tool-info module.